### ISSUE TITLE
my changes to CHORDORG.TXT are ignored, why?

### ISSUE THREAD

**Issue by lushsounds** [TomWhitwell/RM_Chorder#14]
I have the following problem with my new Chord-Organ, which I bought second hand a couple of days ago: No matter what I change in the CHORDORG.TXT file, the module will still play what it has played since I have it. I made a test and made all 16 chords equal, it will still play the default 16 different chords. What am i doing wrong? Thank you in anticipation.

**Comment by Normalised** [comment]
Do you know which firmware is on it?

You can get the latest from here : http://polyfather.com/chord_organ/

**Comment by lushsounds** [comment]
sorry for the late reply, I was away:
No I don't know which firmware is installed.
Is there a known bug regarding my problem in an older version?
Anyhow, I will check if the latest firmware will do it.
Thank you.

**Comment by lushsounds** [comment]
Now I am having difficulties uploading the hex file using Teensy.
The file Chord_Organ_0.6.hex is too large according to a Teensy error message.
Please help!

**Comment by Normalised** [comment]
Can you tell me : 

1. What OS and Version
2. What version of the Teensy Uploader
3. The size (in bytes) of the hex file on your system

**Comment by lushsounds** [comment]
I am using Mac OS X 10.11.6
The Teensy Uploader is version 1.4
The size of the hex file is 200 KB

**Comment by lushsounds** [comment]
ok after my Teensy got blown, I have a new one attached now and everything is working, except:
The Chord Organ's chords work even with a freshly formatted and empty SD card.
I understand that the default chord list and settings are loaded from firmware.
So somehow the file on my SD card gets ignored. What could be the reason?
Please help.

**Comment by TomWhitwell** [comment]
Paging @Normalised - wondering if you've seen this before?

**Comment by Normalised** [comment]
No I haven't. There's a couple of things I would check. 

1. Is the CHORDORG.TXT in the root of the SD card
2. What is in the file?
3. Does Radio Music work?

The RM firmware has a couple of handy features at startup to help debug if the issue is a basic card reading problem or if its the content on the card, if you see the very bottom of this page :

https://github.com/TomWhitwell/RadioMusic/wiki/Radio-Music-2017-Firmware

**Comment by lushsounds** [comment]
thank you @Normalised 
again: 
no CHORDORG.TXT at all on the SD card, but Chord Organ works with the preset chords.

**Comment by Loquelic** [comment]
Hello @Normalised on my case the chord organ doesn't create the  CHORDORG.TXT in the root of the SD card. I installed the latest firmware now but still not working

**Comment by Normalised** [comment]
What about if you put your own CHORDORG.TXT on the card?

If there is nothing on the card then it will use the default settings that are built into the firmware.

If you go to this page : http://polyfather.com/chord_organ/

1. edit the chord settings show in the text box
2. Click DOWNLOAD CONFIG to download the CHORDORG.TXT file to your computer
3. Copy this to the SD card
4. Reboot the module with the SD card and new TXT file.

What happens?

**Comment by TomWhitwell** [comment]
I wonder if the module will work as you describe if the SD card socket is not connected (a missed solder joint) or the SD card itself is faulty. 

The Radio Music firmware is obviously much more dependent on the SD card, so this would normally be identified sooner. Closely check the soldering around the socket - the tiny pins. It’s not a very common problem but worth checking.

**Comment by Loquelic** [comment]
OK, I re-soldered the tiny pins and I can finally see the txt file, but i still can't hear nothing, i don't understand, i can see the leds responds perfectly but without sound, just a noise

**Comment by lushsounds** [comment]
@Normalised: If I put my own CHORDORG.TXT on the card, it gets ignored. So maybe it's what @TomWhitwell says, that the SD card socket is not properly connected. 
The modules behaviour is slighly different though between having no SD card in the socket and having an empty SD card in the socket. With card the module starts almost immediately after powering. Without the card the four lights flash one after the other before the sound comes.

**Comment by Loquelic** [comment]
Yeah it responds as you said, but without sound, I'm missing something

**Comment by Thonk-Steve** [comment]
This may have been covered, but you tweaked the trimmer on the back for
gain level?

**Comment by Loquelic** [comment]
Sorry but i'm not expert ahah, what do you mean?


________________________________
Da: Thonk-Steve <notifications@github.com>
Inviato: mercoledì 2 maggio 2018 23:56
A: TomWhitwell/Chord-Organ
Cc: Loquelic; Comment
Oggetto: Re: [TomWhitwell/Chord-Organ] my changes to CHORDORG.TXT are ignored, why? (#14)

This may have been covered, but you tweaked the trimmer on the back for
gain level?

**Comment by Thonk-Steve** [comment]
get and screwdriver and try adjusting the blue trimmer on the back, it's
just a volume knob.

**Comment by Loquelic** [comment]
yeah i can hear some chord now but the noise is very loud

**Comment by Thonk-Steve** [comment]
is the quiet chord reacting as you'd expect to the controls?

**Comment by TomWhitwell** [comment]
Some Possibilities: 
A missed solder joint 
Two resistor values swapped 
A diode the wrong way around 
A very thin solder bridge somewhere 
A chip the wrong way around 
A grounding problem - sometimes if the RM or Chord Organ is in a separate case from the output, you need a second patch lead between the cases to make a good ground connection

**Comment by Loquelic** [comment]
yeah i can hear any chords when i switch on it, but any time i turn the knob or i push the green bottom i hear a blip sound. I'll take a look at the resistors ecc....

**Comment by lushsounds** [comment]
My problem is resolved.
The SD card was previously ignored, because it wasn't formatted with the option "master boot record". Since I did that (FAT32 + master boot record), the TXT file on the SD card is taken by the module.

**Comment by Thonk-Steve** [comment]
Ah good news!
